tripal_chado/src/Plugin/TripalImporter/GFF3Importer.php

```diff
@@ -343,8 +351,10 @@ public static function create(ContainerInterface $container, array $configuratio
    * {@inheritdoc}
    */
   public function __construct(array $configuration, $plugin_id, $plugin_definition,
-                              ChadoConnection $connection, ChadoBuddyPluginManager $buddy_manager) {
-    parent::__construct($configuration, $plugin_id, $plugin_definition, $connection);
+                              ChadoConnection $connection, TripalBackendPublishManager $publish_manager,
+                              ChadoBuddyPluginManager $buddy_manager) {
```

Comment:
Coding standards update because we changed the line :-)
```suggestion
  public function __construct(
    array $configuration,
    $plugin_id,
    $plugin_definition,
    ChadoConnection $connection,
    TripalBackendPublishManager $publish_manager,
    ChadoBuddyPluginManager $buddy_manager,
  ) {
```

Reply:
Couldn't commit suggestion - done separately